### DISCUSSION TITLE
What switches my CCL to 5 amps and why?

### DISCUSSION THREAD

**Discussion by chadodes** [Louisvdw/dbus-serialbattery#151] [Q&A] [answered]
Hello folks.

Equipment:

- JBD BMS 8s 100a
- Cerbo GX
- USB to TTL to UART (to keep my ve.direct ports available for other devices)
- Multiplus 3K
- BMV-700 (not set to any type of battery monitor, and is not in the DVCC)

In DVCC I have the current limitation set to 40 amps.

Just noticing that the BMS device (CCL) charge current parameters data will flip from 109 amps down to 5 amps.

Is this the driver doing this? is it the BMS it's self?

Thanks!
-Chad

**Comment by Louisvdw** [comment, marked as answer]
The driver reduce the charge current to 5A when you reach 100% SOC. This help to give your balancers some time to work if you have imbalanced cells.
https://github.com/Louisvdw/dbus-serialbattery/wiki/Features#charge-current-control-management
There is a beta feature to disable CCCM by setting CCCM_ENABLE = False in utils.py if you want to disable it.

**Reply by chadodes** [reply]
Thank you sir, and thank you for pointing out the clearly written documentation. This answer's everything.
